instrumentation/grpc/src/it/grpc12/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Maybe not since it's exposed out of tree, but could consider putting these resources in `brave-test`

Reply:
brave-test doesn't currently depend on mockito, and mockito versions have been painful in the past.. good idea in general, but I feel I don't want to move anything mockito related to a wider scope.. make sense?